### ISSUE TITLE
Consider updating to MariaDB 10.6 as better option

### ISSUE THREAD

**Issue by stronk7** [ApacheFriends/xampp-build#17]
Recently (#10) MariaDB was updated to 10.4, that is supported until June 2024.

This is an issue about to consider updating to MariaDB 10.6 instead. Main reason being that it's supported until July 2026 and, also, that it has been declared LTS (while others aren't). Previous LTS was 5.5 that was supported really, really long.

Ref: https://mariadb.com/kb/en/mariadb-server-release-dates/

Ciao :-)

**Comment by grooverdan** [comment]
10.11 has been announced as a [LTS](https://mariadb.org/mariadb-10-11-is-lts/) and this should be out, if all goes well, within a week.

Maintenance policy is https://mariadb.org/about/#maintenance-policy

**Comment by beltran-rubo** [comment]
Thanks! Probably a bit early to move to 10.11 yet but we will work on updating MariaDB component to the LTS version.

**Comment by ralf-krause** [comment]
Please can you tell us when we will get a newer MariaDB?

**Comment by 3xploiton3** [comment]
why not use MariaDB LTS latest version [10.11.7](https://mariadb.com/kb/en/mariadb-10-11-7-changelog/) ? support until (16 Feb 2028)

**Comment by mediaformat** [comment]
Agreed, it is quite surprising to find that xampp is using the _last_ available stable instead of the most recent.

**Comment by grooverdan** [comment]
Note 10.4 goes EOL in 3 months.

**Comment by grooverdan** [comment]
https://mariadb.com/kb/en/mariadb-10-4-34-release-notes/ = officially the last 10.4 release.

**Comment by tranchinh87** [comment]
Django Framework version 5.1.4 is incompatible with the MariaDB versions included in the current XAMPP releases. Please assist us in upgrading to a newer version of MariaDB to ensure compatibility with the latest Django Framework. 
Thank you in advance!

**Comment by tranchinh87** [comment]
> Thanks! Probably a bit early to move to 10.11 yet but we will work on updating MariaDB component to the LTS version.

Django Framework version 5.1.4 is incompatible with the MariaDB versions included in the current XAMPP releases. Please assist us in upgrading to a newer version of MariaDB to ensure compatibility with the latest Django Framework.
Thank you in advance!

**Comment by madiko** [comment]
I am also waiting and hoping for the next release. To keep us update: 

current long-term release:
**MariaDB Server 11.4**
https://mariadb.com/kb/en/mariadb-server-11-4/
maintained until May 2029

current stable rolling release:
**MariaDB Server 11.7**
https://mariadb.com/kb/en/mariadb-server-11-7/

Thank you!

**Comment by grooverdan** [comment]
Also before doing upgrade fixing the stop script otherwise many use cases will fail
https://stackoverflow.com/a/79517757/10195153

11.8 (soon to be released - https://mariadb.org/11-8-is-lts/) is LTS also.
